docker/master/Dockerfile

```diff
@@ -8,7 +10,11 @@ FROM debian:jessie
 RUN groupadd -r freeswitch --gid=999 && useradd -r -g freeswitch --uid=999 freeswitch
 
 # grab gosu for easy step-down from root
-RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4
+RUN apt-get update && apt-get install -y --no-install-recommends dirmngr gnupg2 \
+    && gpg2 --keyserver hkp://keyserver.ubuntu.com --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
```

Comment:
What's the point of this key?

Reply:
It is exist from past

Reply:
@andywolk I think it is used for `gpg --verify /usr/local/bin/gosu.asc`

Reply:
@andywolk I think it is used for `gpg --verify /usr/local/bin/gosu.asc`

Reply:
> @andywolk I think it is used for `gpg --verify /usr/local/bin/gosu.asc`

no